ansible/TestbedProcessing.py

```diff
@@ -416,6 +418,89 @@ def makeLab(data, devices, testbed, outfile):
                                 entry += "\tansible_ssh_pass=" + ansible_ssh_pass
                             except:
                                 print("\t\t" + host + ": ansible_ssh_pass not found")
+                        try: #get hwsku
+                            hwsku = dev.get("hwsku")
+                            if hwsku is not None:
+                               entry += "\thwsku=" + hwsku
+                        except:
+                            print("\t\t" + host + ": hwsku not found")
+
+                        try: #get cardtype
+                            cardtype = dev.get("cardtype")
+                            if cardtype is not None:
+                               entry += "\tcardtype=" + cardtype
```

Comment:
We use 'card_type' as the host variable to distinguish between linecard and supervisor card. Can we change 'cardtype' to 'card_type'.